app/models/host/base.rb

```diff
@@ -162,7 +168,9 @@ def import_facts(facts)
       # If we don't (e.g. we never install the server via Foreman, we populate the fields from facts
       # TODO: if it was installed by Foreman and there is a mismatch,
       # we should probably send out an alert.
-      save(:validate => false)
+      result = save(:validate => false)
+      self.skip_orchestration = false
```

Comment:
I'm thinking you might want to put `self.skip_orchestration = false` on an `ensure` block. 
It's kind of petty but if there's any problem in between lines 161 and this (exceptions etc..), we definitely want to set `skip_orchestration` back to false